tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "module": "commonjs",
     "target": "es6",
     "lib": ["es6"],
-    "sourceMap": true
+    "sourceMap": true,
+    "newLine":  "lf"
```

Comment:
When rebuilding the project locally, CRLF is used. That's a bit painful to maintain as rebuild file is marked as changed, but the only change is LF->CRLF that is not pushed.

See: https://github.com/SonarSource/gh-action-lt-backlog/blob/master/.gitattributes
Docs: https://www.typescriptlang.org/tsconfig#newLine